ahcer/package.json

```diff
@@ -26,6 +26,7 @@
     "angular-csv-ext": "^1.0.5",
     "firebase": "^9.6.7",
     "firebaseui": "^6.0.1",
+    "moment": "^2.29.3",
```

Comment:
So...moment JS is suggesting that we look for alternatives as they are no longer really supported (https://momentjs.com/docs/#/-project-status/) You might reconsider using moment js. We just migrated from moment to [Luxon](https://moment.github.io/luxon/) at Vermeer.